poc/prg.py

```diff
@@ -104,9 +104,12 @@ class PrgSha3(Prg):
     test_vec_name = 'PrgSha3'
 
     def __init__(self, seed, dst, binder):
-        # `dst` is used as the customization string; `seed || binder` is
+        # `dst` is used for domain separation; `seed || binder` is
         # used as the main input string.
-        self.shake = cSHAKE128.new(custom=dst)
+        self.shake = SHAKE128.new()
+        dst_length = to_le_bytes(len(dst), 1)  # one byte value
```

Comment:
nit: This is clear enough from context :)
```suggestion
        dst_length = to_le_bytes(len(dst), 1)
```